server/bin/uninstall.sh

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env bash
+
+remove_enqueuer_wrapper() {
+    local enqueuer=/usr/local/bin/autotest_enqueuer
+    echo "[AUTOTEST-UNINSTALL] removing enqueuer wrapper at ${enqueuer}"
+    sudo rm ${enqueuer}
```

Comment:
I'm going to change all rm to rm -f, in case the file doesn't exist, or if it asks for confirmation with read-only files